Runtime/SpriteSwitchClip.cs

```diff
@@ -0,0 +1,18 @@
+﻿using UnityEngine;
+using UnityEngine.Playables;
+
+namespace Unity.MaterialSwitch
+{
+    public class SpriteSwitchClip : PlayableAsset
```

Comment:
Do we need this to be public ?

Reply:
I'm not sure. It matches the accessability of the corresponding MaterialSwitch classes.

Reply:
Please change to internal as we discussed.